Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRenderer.cs

```diff
@@ -90,6 +90,7 @@ public void RegisterHtmlHeadContents (HtmlHeadAppender htmlHeadAppender)
       var scriptUrl = ResourceUrlFactory.CreateResourceUrl (typeof (BocBooleanValueRenderer), ResourceType.Html, "BocBooleanValue.js");
       htmlHeadAppender.RegisterJavaScriptInclude (scriptFileKey, scriptUrl);
 
+      htmlHeadAppender.RegisterCommonStyleSheet();
```

Comment:
```suggestion
      htmlHeadAppender.RegisterCommonStyleSheet();

```

Reply:
The next code block just deals with registering one stylesheet. addning a blank line structures the code